app/controllers/users_controller.rb

```diff
@@ -12,6 +12,12 @@ def index
   def show
   end
 
+  def current
+    @user = current_user
+
+    render json: { user: @user }
```

Comment:
This style of rendering won't be necessary once you have access to the serializers.
`render json: @user` will implicitly include the root key of user.